### PR TITLE
Locale additions

### PR DESCRIPTION
**Pull request by gregvds**
## Description
Content update for translations in FR, NL and a few fixes in EN.

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)

 - [ x] Bugfix
 - [ x] Feature

**Does this PR introduce a breaking change?** (check one with "x", remove the other)

 - [x ] No

### CODE REVIEW

File: web/client/translations/data.en-US [hunk 1]
```diff
@@ -111,16 +110,6 @@
         "groups": "Groups",
         "permission": "Permission",
         "permissions": "Permissions",
-        "global": {
```
Comment (mbarto) [inline web/client/translations/data.en-US]: You removed stuff that is in use, I think by mistake

File: web/client/translations/data.fr-FR [hunk 2]
```diff
@@ -52,21 +52,20 @@
...
             "elevation": "élévation",
-            "titleTranslations": "Title translations",
-            "groupProperties": "Group properties",
-            "featureInfo": "Feature Info",
-            "featureInfoFormatLbl": "Information, format de la réponse",
-            "legenderror": "La légende n'est pas disponible"
+            "titleTranslations": "Traductions du titre",
+            "groupProperties": "Propriétés du groupe",
+            "featureInfo": "Informations attributaires de l'objet",
+            "featureInfoFormatLbl": "Information, format de la réponse"
```
Comment (mbarto) [inline web/client/translations/data.fr-FR]: legenderror is missing

File: web/client/translations/data.fr-FR [hunk 3]
```diff
@@ -109,26 +108,16 @@
         "confirmTitle": "Confirmez-vous?",
         "pageInfo": "{total, plural, =0 {aucun objet} =1 {{total} objet parmi {total}} other {objets {start}-{end} parmi {total}}}",
         "loading": "Chargement...",
-        "group": "Group",
-        "groups": "Groups",
+        "group": "Groupe",
+        "groups": "Groupes",
         "permission": "Permission",
         "permissions": "Permissions",
-        "global": {
```
Comment (mbarto) [inline web/client/translations/data.fr-FR]: Restore this block, please

File: web/client/translations/data.fr-FR [hunk 4]
```diff
@@ -237,31 +226,29 @@
...
+            "noFilteredResults": "Pas de résultats",
+            "filterPlaceholder": "Filtrer les couches",
+            "clearFilter": "Supprimer le filtre",
+            "toolReloadLayerTooltip": "Forcer la recharge de la couche sélectionnée",
+            "toolReloadLayersTooltip": "Forcer la recharge des couches sélectionnées",
+            "statusIconOpen": "Fermer le groupe de couches",
+            "statusIconClose": Ouvrir le groupe de couches",
+            "grabLayerIcon": "Attraper et déplacer la couche",
+            "grabGroupIcon": "Attraper et déplacer le groupe",
+            "toggleLayerVisibilityWarning": "Inverser la visibilité de la couche, attention: la couche ne s'est pas chargée correctement",
```
Comment (mbarto) [inline web/client/translations/data.fr-FR]: createWidget is missing

File: web/client/translations/data.fr-FR [hunk 5]
```diff
@@ -967,108 +954,6 @@
                 "invalidOutputFormat": "Le format d'exportation sélectionné n'est pas disponible"
             }
         },
-        "widgets": {
```
Comment (mbarto) [inline web/client/translations/data.fr-FR]: You removed stuff that is in use, I think by mistake

**Comment by gregvds** [comment]
Added the missing lines (I hope I didn't forget something).

File: web/client/translations/data.fr-FR (hunk 4 above)
Comment (gregvds) [reply web/client/translations/data.fr-FR]: Added

**Comment by gregvds** [comment]
Thanks again for the quick and kind help.
Hope to have it right now :-).

**Comment by mbarto** [comment]
Looks good, merging, thanks

**Comment by gregvds** [comment]
Great :-).

Thanks a lot!

**Pull request merged**